scripts/toolboxSrc/postDeploy.ts

```diff
@@ -6,10 +6,6 @@ import getProjectRoot from '../webpack/utils/getProjectRoot'
 
 const PROJECT_ROOT = getProjectRoot()
 
-const flushSocketConnections = async () => {
```

Comment:
As we're no longer storing `connectedSockets` in User, I've removed this. We could flush all connected sockets in Redis but as this is already happening in disconnect, I don't think that's necessary?

Reply:
what happens when we deploy a new version? the old server shuts down without callings disconnect.